### DISCUSSION TITLE
New forum - test

### DISCUSSION THREAD

**Discussion by JoseRamosMendez** [topas-nbio/TOPAS-nBio#1] [Announcements]
First message in OpenTopas forum @jschuemann

**Comment by jschuemann** [comment]
But can I reply even though it's closed?

**Comment by JoseRamosMendez** [comment]
Yes, it seems you can

From: jschuemann ***@***.***>
Date: Thursday, November 30, 2023 at 9:22 AM
To: OpenTOPAS/TOPAS-nBio ***@***.***>
Cc: JoseRamosMendez ***@***.***>, State change ***@***.***>
Subject: Re: [OpenTOPAS/TOPAS-nBio] New forum - test (Discussion #1)

But can I reply even though it's closed?

**Reply by jschuemann** [reply]
So it's only partially what we want.
